injector/dns_disruption.go

```diff
@@ -155,8 +155,12 @@ func (i DNSDisruptionInjector) Inject() error {
 	return nil
 }
 
+func (i *DNSDisruptionInjector) UpdateConfig(config Config) {
```

Comment:
Is the reason all the receivers in the injector package changed to pointers because of updateConfig?

Reply:
Yes, I'm not sure that's the right move